api/src/identity-access-management/application/token/token.controller.js

```diff
@@ -26,37 +26,41 @@ const createToken = async function (request, h, dependencies = { tokenService })
   let accessToken, refreshToken;
   let expirationDelaySeconds;
 
-  if (request.payload.grant_type === 'refresh_token') {
+  const grantType = request.payload.grant_type;
+  const scope = request.payload.scope;
+
+  if (grantType === 'refresh_token') {
     refreshToken = request.payload.refresh_token;
-    const accessTokenAndExpirationDelaySeconds = await usecases.createAccessTokenFromRefreshToken({ refreshToken });
-    accessToken = accessTokenAndExpirationDelaySeconds.accessToken;
-    expirationDelaySeconds = accessTokenAndExpirationDelaySeconds.expirationDelaySeconds;
-  } else if (request.payload.grant_type === 'password') {
+
+    // TODO: we should pass the scope when ember-simple-auth will pass it
```

Comment:
Suggestion: 
Ajouter l'url de la PR sur ember-simple-auth https://github.com/mainmatter/ember-simple-auth/pull/2813 pour les détails.
```suggestion
    // TODO: we should pass the scope when ember-simple-auth will pass it
    // see https://github.com/mainmatter/ember-simple-auth/pull/2813 for further details
```

Reply:
@er-lim tous les commentaires pris en compte